CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 Voucherify Java SDK - Changelog
 ===============================
+- **2021-12-21** - `8.1.0` - Add support for Session on Voucher validation & redemption as well as session release
```

Comment:
please remove this as the most recent changelog is available in `readme.md`, this file acts as a history changelog (to not add too much noise in readme.md)

Reply:
Changes reverted